assets/js/form.js

```diff
@@ -9,55 +9,63 @@ function validation() {
 }
 
 function displayMessage() {
-  let emailForm = document.getElementById("emailFormContainer");
-  let formMessage = document.getElementById("formMessage");
+  let emailForm = document.getElementById('emailFormContainer');
+  let formMessage = document.getElementById('formMessage');
 
-  emailForm.className += " hidden";
-  formMessage.className += "active";
+  console.log('formMessage: ', formMessage);
```

Comment:
Can we remove this `console.log`? (also in other places). Thanks. cc. @MB-Designs